### PR TITLE
Support for Catalina

### PR DESCRIPTION
**Pull request by Plopix**
https://github.com/ezsystems/launchpad/issues/68

### CODE REVIEW

**Comment by Webarkitekt** [comment]
Seems ok to me

File: .travis.yml [hunk 1]
```diff
@@ -12,12 +12,10 @@ cache:
 
 php:
   - 5.6
-  - 7.0
-  - 7.1
```
Comment (andrerom) [inline .travis.yml]: irrelevant change?

File: .travis.yml [hunk 2]
```diff
@@ -12,12 +12,10 @@ cache:
...
   - 7.2
+  - 7.3
 
 env:
-  - TEST_COLLECTION=collection-1x.json EZVERSION=1.13.*
```
Comment (andrerom) [inline .travis.yml]: same here?

Comment (Plopix) [reply .travis.yml]: same here, I guessed we don't really care anymore of 1.x

File: .travis.yml (hunk 1 above)
Comment (Plopix) [reply .travis.yml]: In fact the Travis tests were failing for eZ 1.x, I guessed we could remove that.

File: .travis.yml (hunk 2 above)
Comment (andrerom) [reply .travis.yml]: We'll that is up to you. Any insight into why it fails?

Comment (andrerom) [reply .travis.yml]: UPDATE all PHP versions fail with `Warning: "continue" targeting switch is equivalent to "break`

So main issue on your side is that they all pull in PHP 7.3:
`Status: Downloaded newer image for plopix/docker-php-ez-engine:7.3`

And perhaps on eZ side or something there is a PHP 7.3 incompatibility as well.

Comment (Plopix) [reply .travis.yml]: hum yes that would make sense, I enforce PHP 7.3 everywhere that is prob why it is failing...

Comment (andrerom) [reply .travis.yml]: After some digging into it remember it now, it's due to us having to ship with composer.lock generated on minimum supported PHP versions.

And packages like `zendframework/zend-code` as well as packages by ocramius are made per PHP version, and they thus don't bother fixing incompatibilities like this in older versions.

So as this is not something we can fix, either:
- don't install with PHP 7.3 on 1.x in the first place
- Or run create-project with `--no-scripts` and then do a `composer update` to update packages for PHP 7.3

**Comment by Plopix** [comment]
Thank you @andrerom!  You kind of implicitly forced me not to be lazy and take a dirty shortcut!

We are back on the same tests as before and all green ;-)

I will merge soon!

**Pull request merged**